unreleased_history/bug_fixes/fix_some_write_perf_statistics.md

```diff
@@ -0,0 +1 @@
+Fix some perf context statistics error in write steps
```

Comment:
Could you spell out the statistics that you changed in the change log? Thanks.

Reply:
ok, I have added some detailed descriptions